### ISSUE TITLE
Hang when editing commit message in tortoise git

### ISSUE THREAD

**Issue by RichCaloggero** [nvaccess/nvda#8882]
### Steps to reproduce:
1. start a commit in tortoise git
2. type two characters (say "x" and "y")
3. press left arrow and hear "y" spoken
4. press left arrow again and NVDA hangs

In most recent version (2018.3.2) the hang is aprox. 10 or 20 seconds, after which time nvda becomes responsive again. Previous version tested (2018.2.1) would hang possibly forever.

Testing with Jaws-2018 does not exhibit this behavior. Tortoise git developer cannot reproduce.

### Actual behavior:

NVDA hangs (see above).

### Expected behavior:

The character "x" should be spoken.

### System configuration:

win7 64-bit.

#### NVDA Installed/portable/running from source:

NVDA installed

#### NVDA version:
- 2018.3.2
- 2018.2.1

Would like to test older releases, but cannot find binaries.

#### Name and version of other software in use when reproducing the issue:

tortoise git v2.7.0.0

#### Does the issue still occur after restarting your PC?

Yes.

#### Have you tried any other versions of NVDA?

Yes, see above.

**Comment by Brian1Gaff** [comment]
I agree old versions should have a direct link to their page from somewhere 
obvious on the web site.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by DrSooom** [comment]
Download-Link to NVDA 2018.1: https://www.nvaccess.org/download/?nvdaVersion=2018.1

Sadly https://www.nvaccess.org/files/nvda/releases/ is forbidden (error 403).

**Comment by leonardder** [comment]
@RichCaloggero: could you please test using NVDA 2018.1?

Download-Link to NVDA 2018.1.1: https://www.nvaccess.org/download/?nvdaVersion=2018.1.1

**Comment by RichCaloggero** [comment]
I've just tried two different older releases with slightly different, but similarly buggy results.

Note that completely removing my appData NVDA folder makes no difference!

### 2018.1.1:

- open commit dialog and type "xy"
- leftArrow: it says "y"
- leftArrow: it hangs
  + about every 10 or 15 seconds, reads title bar, but is still hung
  - after three of these cycles, it becomes unhung and I can hit rightArrow and hear "y"

### 2017.2:

- same initial conditions
- when it hangs, it hangs for about 20 seconds, and then says "blank", at which point it is unhung


Hope this is helpful!

**Comment by dnz3d4c** [comment]
This problem can also be reproduced in the latest NVDA alpha version.
To avoid this problem now, copy and paste the commit message.

**Comment by tmthywynn8** [comment]
I think this should be fixed in ##9612. If memory serves, when committing messages in Tortoise Git, the Scintilla editor is also used, so the referenced pull request may also fix this issue as well.

**Comment by Adriani90** [comment]
I cannot reproduce this issue in NVDA 2019.2 Beta 1.
@dnz3d4c, @RichCaloggero are you still having this problem?

**Comment by Adriani90** [comment]
I am closing this as fixed and works for me. Please comment on it if this is not correct and we can reopen it. Thanks.
